qutebrowser/config/qtargs.py

```diff
@@ -324,13 +335,37 @@ def _qtwebengine_args(
         'auto':
             '--enable-experimental-web-platform-features' if machinery.IS_QT5 else None,
     },
+    'qt.workarounds.disable_accelerated_2d_canvas': {
+        'always': '--disable-accelerated-2d-canvas',
+        'never': None,
+        'auto': lambda versions, namespace, special_flags: 'always'
+        if machinery.IS_QT6
+        and versions.chromium_major
+        and versions.chromium_major < 111
+        else 'never',
+    },
 }
 
 
-def _qtwebengine_settings_args() -> Iterator[str]:
+def _qtwebengine_settings_args(
+    versions: version.WebEngineVersions,
+    namespace: argparse.Namespace,
+    special_flags: Sequence[str],
+) -> Iterator[str]:
     for setting, args in sorted(_WEBENGINE_SETTINGS.items()):
         arg = args[config.instance.get(setting)]
-        if arg is not None:
+        if callable(arg):
```

Comment:
Seems like a bit of a hack... but okay, I can't think of anything that's just as simple, so I suppose it's okay.